sound/soc/sof/topology.c

```diff
@@ -962,6 +985,7 @@ static int sof_control_load_volume(struct snd_soc_component *scomp,
 		container_of(hdr, struct snd_soc_tplg_mixer_control, hdr);
 	struct sof_ipc_ctrl_data *cdata;
 	int tlv[TLV_ITEMS];
+	int read_bytes = 0;
```

Comment:
you initialise `*read_bytes` inside `sof_parse_tokens()`, so, no need to also initialise it in each caller function

Reply:
ok